src/app.css

```diff
@@ -11,6 +11,7 @@
 		local(''),
 		url('/Hershey-Noailles-Futura-Duplex-Regular.woff') format('woff'),
 		url('/Hershey-Noailles-Futura-Duplex-Regular.ttf') format('truetype');
+	font-display: swap;
```

Comment:
by default, `font-block` is used which blocks rendering until the fonts are loaded. `font-swap` renders the text with the fallback font and then swaps them to the custom one.